examples/tactics/Postprocess.fst

```diff
@@ -64,11 +64,18 @@ let xx = C1 (function
 
 open FStar.FunctionalExtensionality
 
+let q_as_lem (#a:Type) (#b:a -> Type) (p:squash (forall x. b x)) (x:a)
+  : Lemma (b x)
+  = ()
+
+let congruence_fun #a (#b:a -> Type) (f g:(x:a -> b x)) (x:squash (forall x. f x == g x)) :
+  Lemma (ensures (fun (x:a) -> f x) == (fun (x:a) -> g x)) =
+  assert ((fun (x:a) -> f x) == (fun (x:a) -> g x))
+      by (l_to_r [quote (q_as_lem x)];
+          trefl())
```

Comment:
Here's a hand-rolled use of functional extensionality. We didn't need to use the on_domain stuff on restricted functions from FStar.FunctionalExtensionality